tests/__init__.py

```diff
@@ -7,14 +7,42 @@
 
 from sqlalchemy_defaults import make_lazy_configured
 
+IS_MYSQL = (
+    os.environ.get('DB') == 'mysql' or
+    os.environ.get('DSN', '').startswith('mysql')
+)
+MYSQLD_VERSION = tuple(
```

Comment:
We don't have a connection/dialect object this early in the tests lifecycle. Perhaps one could make them into fixtures, but the way the tests are organized now does not exactly make it easy without completely refactoring them.